### DISCUSSION TITLE
Welcome to Autonomous Driving in Carla using DRL Discussions!

### DISCUSSION THREAD

**Discussion by idreesshaikh** [idreesshaikh/Autonomous-Driving-in-Carla-using-Deep-Reinforcement-Learning#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by satyam7010** [comment]
Hi Idrees, i was reading your work on autonomous car using RL, and i am  just curious how much time take to train this model and what configurations you used?

**Comment by AKG47-Auto** [comment]
Hi Idrees, is there any way to use the newer carla versions for your project?
